examples/pdf-service.nrm

```diff
@@ -1,3 +1,5 @@
+@error("")
```

Comment:
Annotations don't have to have arguments (including parentheses).  Therefore we can simply this as `@error`.  See also the [parser code][1].

[1]: https://github.com/spoqa/nirum/blob/cb59c832dd7f6f09debdcec6934ecc80fb223e14/src/Nirum/Parser.hs#L161-L168